tests/remote_attestation/maa/README.md

```diff
@@ -0,0 +1,26 @@
+Test Scenario: Remote Attestation with MAA
+==========================================
+In this test scenario we have two enclaves: sgxlkl-enclave and oe-enclave.
+sgxlkl-enclave app attests oe-enclave app via MAA (Microsoft Azure Attestation service) 
+
+Here is the test flow:
+1) oe-enclave is the tls server and runs and sleeps 45 seconds and exits
+2) sgxlkl-enclave starts and connects to oe-enclave via TLS trusted connection
+3) sgxlkl-enclave gets report of oe-enclave at TLS connection
+3) sgxlkl-enclave gets authentication token from AAD (Azure Active Directory) in order to access MAA
+4) sgxlkl-enclave calls MAA remote attestation service for JWT token and gets, parses and verifies
+
+This scenario test depends on passed environment variables for credentials and urls accessing MAA service. 
+These are injected by ADO in pipeline:
+- MAA_CLIENT_ID,
```

Comment:
Could you add a short sentence on what each of them is? This has to be reproducible by any external developer.

Reply:
Will do that